front/src/Turtle/index.js

```diff
@@ -0,0 +1,109 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Dropdown from "../Dropdown";
+import Battery from "../Battery";
+import styled from "styled-components";
+
+const TEAM_VALUES = ["Magenta", "Cyan"];
+const HOME_VALUES = ["Yellow home", "Blue home"];
+const ROLE_VALUES = [
+  "INACTIVE",
+  "ROLE_NONE",
+  "Att main",
+  "Att assist",
+  "Def main",
+  "Def assist",
+  "Def assist 2",
+  "Goalkeeper"
+];
+
+/**
+ * Show all details for a turtle
+ * Author: H.E. van der Laan
+ *
+ * Props:
+ * - turtle: An object containing the following turtle details:
+ *   - battery: the current battery status of the turtle
+ *   - home: the current home goal of this robot
+ *   - id: the ID number of the turtle
+ *   - role: the current role of this robot
+ *   - team: the current team of this robot
+ * - editable: whether this turtle's properties can be edited
+ * - onChange: a function with two arguments that is called when one of the dropdowns is changed. The first argument is name of the prop that is changed, the second argument is its new value.
+ */
+const Turtle = props => {
+  const { battery, home, id, role, team } = props.turtle;
+  const { editable, onChange } = props;
+  return (
+    <DefaultTurtle>
+      <Battery percentage={battery} />
+      <SubSection>
```

Comment:
Do you really need this `div` ?

Reply:
No, but there might be additional indicators in the middle. We actually plan on doing that, but these indicators weren't in the current design